### PR TITLE
Documentation for getting list of recently used shared printers

### PR DESCRIPTION
**Pull request by mayankbansal018**

### CODE REVIEW

**Comment by Lauragra** [comment]
@mayankbansal018 , please follow the [Get started](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211776/Get-started-with-GitHub) steps to set yourself up to be a contributor to the docs. This PR is missing some required elements such as a changelog entry and a TOC update.

**Comment by mayankbansal018** [comment]
> @mayankbansal018 , please follow the [Get started](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211776/Get-started-with-GitHub) steps to set yourself up to be a contributor to the docs. This PR is missing some required elements such as a changelog entry and a TOC update.

@Lauragra can you please review this PR again

**Comment by Lauragra** [comment]
> > @mayankbansal018 , please follow the [Get started](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211776/Get-started-with-GitHub) steps to set yourself up to be a contributor to the docs. This PR is missing some required elements such as a changelog entry and a TOC update.
> 
> @Lauragra can you please review this PR again

I haven't reviewed the PR yet; before I can do that, I need a pointer to your roadmap item so I can view the associated api.md and CSDL files. When you're ready for us to review your PR, please set the **ready for content review** label. You need to be a member of the contributor's group to do that, so be sure to follow the Get started steps.

**Comment by mayankbansal018** [comment]
> > > @mayankbansal018 , please follow the [Get started](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211776/Get-started-with-GitHub) steps to set yourself up to be a contributor to the docs. This PR is missing some required elements such as a changelog entry and a TOC update.
> > 
> > 
> > @Lauragra can you please review this PR again
> 
> I haven't reviewed the PR yet; before I can do that, I need a pointer to your roadmap item so I can view the associated api.md and CSDL files. When you're ready for us to review your PR, please set the **ready for content review** label. You need to be a member of the contributor's group to do that, so be sure to follow the Get started steps.

I'm not sure about the roadmap document, however here's link to api.md https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding?path=%2Freviews%2F2123-universal-print-ms-graph-api%2Fapi.md&version=GBmaster&_a=preview

**Comment by Lauragra** [comment]
> > > > @mayankbansal018 , please follow the [Get started](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211776/Get-started-with-GitHub) steps to set yourself up to be a contributor to the docs. This PR is missing some required elements such as a changelog entry and a TOC update.
> > > 
> > > 
> > > @Lauragra can you please review this PR again
> > 
> > 
> > I haven't reviewed the PR yet; before I can do that, I need a pointer to your roadmap item so I can view the associated api.md and CSDL files. When you're ready for us to review your PR, please set the **ready for content review** label. You need to be a member of the contributor's group to do that, so be sure to follow the Get started steps.
> 
> I'm not sure about the roadmap document, however here's link to api.md https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding?path=%2Freviews%2F2123-universal-print-ms-graph-api%2Fapi.md&version=GBmaster&_a=preview

Thanks; please also make sure that you have an item in the roadmap for tracking purposes. For details, please see the CDK documentation.

**Comment by Lauragra** [comment]
@mayankbansal018 , is this PR ready for us to review? If so, please set the ready for content review label to add it to the queue.

**Comment by mayankbansal018** [comment]
@Lauragra I've forwarded this thread to my previous team to take this PR to completion.

File: changelog/Microsoft.PrintService.json [hunk 1]
```diff
@@ -1,5 +1,49 @@
...
+      "Cloud": "Prod",
+      "Version": "beta",
+      "CreatedDateTime": "2022-03-11T03:23:57.6570047Z",
+      "WorkloadArea": "Devices and apps",
+      "SubArea": "Cloud printing"
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "ae312013-2c53-4ae2-b3ef-1107518c38c1",
```
Comment (JarbasHorst) [inline changelog/Microsoft.PrintService.json:32]: @mayankbansal018, this change is not part of you PR. If the print property should be added to the user resource, please apply this change to the user resource topic.

File: api-reference/beta/api/print-list-recentshares.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+---
+title: Get recent shares
```
Comment (JarbasHorst) [inline api-reference/beta/api/print-list-recentshares.md:2]: What is the corresponding resource topic for this API, printerShare or userPrint? I wonder if there is a relationship between this API topic and the schema below:

image

**Comment by JarbasHorst** [comment]
Hello @mayankbansal018, I reviewed your PR and left some comments. Please, take a look when you will have time.

Also, to avoid merge conflicts, please merge PR [1](https://github.com/mayankbansal018/microsoft-graph-docs/pull/1), [2](https://github.com/mayankbansal018/microsoft-graph-docs/pull/2), [3](https://github.com/mayankbansal018/microsoft-graph-docs/pull/3), [4](https://github.com/mayankbansal018/microsoft-graph-docs/pull/4), and [5](https://github.com/mayankbansal018/microsoft-graph-docs/pull/5) into your branch to include my fixes and before addressing my comments.

File: api-reference/beta/api/print-list-recentshares.md (hunk 2 above)
Comment (nilakhan) [reply api-reference/beta/api/print-list-recentshares.md:2]: @JarbasHorst  The corresponding resourceTopic should be printerShare.

Comment (Jarbas-MSFT) [reply api-reference/beta/api/print-list-recentshares.md]: Thanks. In this case, I will add the List recentprinterShare method to the printerShare resource topic.

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review. According to the author, List recentPrinterShares belongs to the printerShare resource topic. I wonder if this method has a relationship with the printer resource. For example, should List recentPrinterShares be added as a method under the printer resource topic?

**Review by Lauragra — APPROVED** [review]

**Pull request merged**